mne/viz/topomap.py

```diff
@@ -1126,7 +1126,9 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
         times = _process_times(evoked, times, n_peaks=len(axes))
     else:
         times = _process_times(evoked, times, n_peaks=None)
-
+    if max(times) > evoked.times[-1] or min(times) < evoked.times[0]:
+        raise ValueError('Times should be between {0:0.3f} and '
+                         '{1:0.3f}.'.format(evoked.times[0], evoked.times[-1]))
```

Comment:
I suspect this will break something that it shouldn't. For example with MGH sample rate I think you can set a `tmin` in between two samples that's closer to sample above it than below it, then if you pass that `tmin` in as a time, it will throw an error here that it wouldn't before. Is this why you had to modify the test above?